### DISCUSSION TITLE
mc4 cheat doesn't work

### DISCUSSION THREAD

**Discussion by youchiro23** [GoldHEN/GoldHEN_Cheat_Repository#88] [General]
I have the zombie army trilogy 1.01 with cheat in mc4 format but it doesn't work I'm using goldhen 2.3 in cfw 9.00 but when I press the share button it doesn't detect the cheat

**Comment by UnsealedWind** [comment]
Open cheat manger and select your file and then select converter to xml it worked with me that way
